app/templates/editions/octane.hbs

```diff
@@ -24,6 +24,140 @@
   <li><a href="https://emberjs.github.io/rfcs/0364-roadmap-2018.html">Read The Roadmap RFC</a></li>
 </ul>
 
+<h2>Are you adventurous and want to test each of the features out?</h2>
+<p>
+  All features of Octane are presently landed in the latest version of ember,
+  but if you'd like to try pieces of Octane before upgrading all the way to 3.14+,
+  you may find the below helper: a quick reference of each of the features, and when you can use them.
+</p>
+
+<table class="octane-feature-summary-table">
+  <thead>
+    <tr>
+      <th>Ember Version</th>
+      <th>Feature</th>
+      <th>Polyfill</th>
+      <th class="notes-column">Notes</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td rowspan="3"><a href="https://blog.emberjs.com/2018/04/13/ember-3-1-released.html">3.1</a></td>
+      <td>Native Getters</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>Named Arguments</td>
+      <td></td>
+      <td>@foo vs foo vs this.foo</td>
+    </tr>
+    <tr>
+      <td>Optional Features: Application Template Wrapper &amp; Temlpate-only Glimmer Compnonents</td>
+      <td></td>
+      <td>Introduced with defaults: "off"</td>
+    </tr>
+    <tr class="spacer"><td colspan="4"></td></tr>
```

Comment:
```suggestion
    <tr class="spacer"><td colspan="4"></td></tr>
```
Can you explain more about what this is?

Reply:
a way to visually separate the different versions

Reply:
This will mess up screen readers, so let's add something with CSS instead.